src/util/ec.rs

```diff
@@ -11,11 +11,9 @@
 // If not, see <http://creativecommons.org/publicdomain/zero/1.0/>.
 //
 
-//! ECDSA Bitcoin keys.
-//!
-//! This module provides ECDSA keys used in Bitcoin that can be roundtrip
-//! (de)serialized.
+//! Elliptic curve Bitcoin keys and signatures.
 //!
+//! This module provides compressed and uncompressed Bitcoin keys and ECDSA signature-related types.
```

Comment:
In f94fb6dde81e28aacd268b96d49ae30a9427c664:

I don't like this rename. The implication is that xonly keys are not EC keys. They should be called "full keys" or something like that. Honestly I think the entire contents of util/ecdsa.rs could just be pulled into key.rs directly.